string object_name
geometry_msgs/PoseStamped target_pose
---
#result definition
# Comes from moveit_msgs/MoveItErrorCodes
int32 error_code
---
#feedback
